src/core/display/ObservablePoint.js

```diff
@@ -71,5 +72,5 @@ ObservablePoint.prototype.set = function (x, y)
     this._x = x || 0;
     this._y = y || ( (y !== 0) ? this._x : 0 );
 
-    this.transform._versionLocal++;
+    this.transform._versionLocal++; // TODO: Pretty sure this doesn't exist.
```

Comment:
@ivanpopelyshev pinged me and told me there's a fix in a PR he has out, so that TODO can be ignored.